### PR TITLE
refactor: cleanup module.cpp

### PR DESCRIPTION
**Pull request by ice0**

### CODE REVIEW

File: synfig-core/src/synfig/module.cpp [hunk 1]
```diff
@@ -80,51 +71,46 @@ Module::subsys_init(const String &prefix)
...
-	book_=new Book;
+
 	return true;
 }
 
 bool
-Module::subsys_stop()
+synfig::Module::subsys_stop()
 {
-	delete book_;
```
Comment (rodolforg) [inline synfig-core/src/synfig/module.cpp:118]: `book_.clear()`?

Comment (ice0) [reply synfig-core/src/synfig/module.cpp:118]: Yeah, I thought about it. But `synfig::Module` is a singleton-like object, and `book_` is a static variable, so I'm not sure it's necessary.

**Review by rodolforg — APPROVED** [review]
I thought that `USE_CF_BUNDLES` was somehow related to MacOS...

**Comment by ice0** [comment]
> I thought that `USE_CF_BUNDLES` was somehow related to MacOS...

Same for me, but Google shows only our code for this keyword, so I just removed it.

**Comment by rodolforg** [comment]
I just did a `git grep -iE "CF_?BUND"` and files:

- autobuild/osx/app-template/Contents/Info.plist
- synfig-osx/launcher/Synfig Studio.pbproj/project.pbxproj

have some stuff related to `<key>CFBundle[...]`
I don't understand anything about it, but you know, own and use MacOs. So, approved ;)

**Pull request merged**